mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -158,6 +175,413 @@ properties:
+                            required: true
+                            description: |
+                              Likelihood adjustment to apply to all matching findings.
+                            properties:
+                              - !ruby/object:Api::Type::Enum
+                                name: 'fixedLikelihood'
+                                description: |
+                                  Set the likelihood of a finding to a fixed value. Either this or relative_likelihood can be set.
+                                values:
+                                  - :VERY_UNLIKELY
+                                  - :UNLIKELY
+                                  - :POSSIBLE
+                                  - :LIKELY
+                                  - :VERY_LIKELY
+                              - !ruby/object:Api::Type::Integer
+                                name: 'relativeLikelihood'
+                                description: |
+                                  Increase or decrease the likelihood by the specified number of levels. For example,
+                                  if a finding would be POSSIBLE without the detection rule and relativeLikelihood is 1,
+                                  then it is upgraded to LIKELY, while a value of -1 would downgrade it to UNLIKELY.
+                                  Likelihood may never drop below VERY_UNLIKELY or exceed VERY_LIKELY, so applying an
+                                  adjustment of 1 followed by an adjustment of -1 when base likelihood is VERY_LIKELY
+                                  will result in a final likelihood of LIKELY. Either this or fixed_likelihood can be set.
+                      - !ruby/object:Api::Type::NestedObject
+                        name: 'exclusionRule'
+                        description: The rule that specifies conditions when findings
+                          of infoTypes specified in InspectionRuleSet are removed from
+                          results.
+                        properties:
+                          - !ruby/object:Api::Type::Enum
+                            name: 'matchingType'
+                            required: true
+                            description: |
+                              How the rule is applied. See the documentation for more information: https://cloud.google.com/dlp/docs/reference/rest/v2/InspectConfig#MatchingType
+                            values:
+                              - :MATCHING_TYPE_FULL_MATCH
+                              - :MATCHING_TYPE_PARTIAL_MATCH
+                              - :MATCHING_TYPE_INVERSE_MATCH
+                          - !ruby/object:Api::Type::NestedObject
+                            name: 'dictionary'
+                            description: Dictionary which defines the rule.
+                            properties:
+                              - !ruby/object:Api::Type::NestedObject
+                                name: 'wordList'
```

Comment:
exactly_one_of is needed to added to those two fields, wordList and cloudStoragePath

Reply:
Currently, `ConflictsWith`, `ExactlyOneOf` or `AtLeastOneOf` are set at the attribute level, which means we can't set it on a list if we have a list of objects, and each object has a list of attributes that's `AtLeastOneOf`. We don't know the length of the list and there is no way to differentiate object[0] from object[1] with what is conflicting. Hence, the use of the above mentioned functions when the attributes are part of array won't work.